src/mission_companion.cpp

```diff
@@ -400,7 +409,7 @@ bool talk_function::outpost_missions( npc &p, const std::string &id, const std::
             "> Rots in < 2 days: 60%%\n"
             "> Rots in < 5 days: 80%%\n \n"
             "Total faction food stock: %d kcal or %d day's rations", 10 * camp_food_supply(), camp_food_supply( 0, true ) );
-        mission_key_push( key_vectors, "Distribute Food" );
+        mission_key_push( mission_key_vectors, "Distribute Food", _("Distribute Food") );
 
         col_missions["Reset Sort Points"] = string_format("Notes:\n"
```

Comment:
Here, and many more after it as well. You should probably use search function to find and replace all these.

Reply:
How are you requesting these be formatted?  I'm assuming sentence structure is also going to be different, which means using the string_format structure itself isn't going to work.  There isn't a way to push the entire string_format() block to json, is there?

Reply:
Emm, I dont know much about coding, but could you follow npcs/missiondef.json and src/missiondef.cpp ?

So, the json object is loaded by mission_type::load() so you may need to modified it to accept your data format.